easybuild/easyblocks/generic/cmakemake.py

```diff
@@ -51,9 +53,20 @@ def extra_options(extra_vars=None):
         extra_vars.update({
             'srcdir': [None, "Source directory location to provide to cmake command", CUSTOM],
             'separate_build_dir': [False, "Perform build in a separate directory", CUSTOM],
+            'buildtype': [CMAKE_BUILD_TYPES[0], "Build type for CMake. Accepted values: " + ', '.join(CMAKE_BUILD_TYPES),
```

Comment:
I don't like this approach, we might miss the impact of reordering the `CMAKE_BUILD_TYPES` alphabetically

I prefer using `CMAKE_BUILD_TYPE_RELEASE`...